ironfish/src/rpc/routes/wallet/getBalances.ts

```diff
@@ -9,6 +9,7 @@ import { getAccount } from './utils'
 export interface GetBalancesRequest {
   account?: string
   confirmations?: number
+  ownerOnly?: boolean
```

Comment:
I think this is a good improvement, but If you don't mind changing it up, I'd prefer doing the filtering in the CLI rather than adding filtering to the request, since we haven't established a pattern for doing that in RPC calls yet.

Basically this would mean:
* `getBalances` should return an `assetOwner` with each balance
* in `utils\asset.ts`, there should be a check at the top of the function similar to `options.showNativeAsset` that fetches the public address for an account, then filters `balances` by `assetOwner` using that public address.

Reply:
ok, updated to return `assetOwner` with getBalances rpc. renamed utils/asset.ts flag to `option.showNonOwnerAsset`.